src/observer/models.py

```diff
@@ -345,3 +354,33 @@ def __str__(self):
 
     def __repr__(self):
         return '<Content "%s">' % self
+
+
+# ---
+
+def content_link(content):
+    # todo: pad feature
+    path_map = {
+        INTERVIEW: "interviews/{id}",
+        COLLECTION: "collections/{id}",
+        BLOG_ARTICLE: "inside-elife/{id}",
+        FEATURE: "articles/{id}",
+        EDITORIAL: "articles/{id}",
+        INSIGHT: "articles/{id}",
+        DIGEST: "digests/{id}",
+        LABS_POST: "labs/{id}",
+        PODCAST: "podcast/episode{id}",
+    }
+    assert content.content_type in path_map, "cannot find path to content for content type %r" % content.content_type
+    return path_map[content.content_type].format(id=content.id)
+
+def content_url(content):
```

Comment:
push to https://docs.djangoproject.com/en/2.2/ref/models/instances/#django.db.models.Model.get_absolute_url